src/Cake.Web.Core/Content/Blog/BlogReader.cs

```diff
@@ -58,10 +58,12 @@ public BlogIndex Parse(DirectoryPath path)
                     // This is kind of a hack, but we need to make sure that all links are absolute.
                     var feedBody = RewriteRelativeLinks(body);
 
+                    var author = content.GetFrontMatter("author");
+
                     // Add the blog post.
                     posts.Add(new BlogPost(filename.Slug,
                         content.GetFrontMatter("title"), body, feedBody, excerpts, filename.PostedAt,
-                        GetCategories(content)));
+                        GetCategories(content), author == null ? "n/a" : author));
```

Comment:
Perhaps we can change `n/a` to something like `Cake Team`. This will make a better fallback value if we forget to set author in the future.